tests/modules/submissions/resources/test_permissions.py

```diff
@@ -7,21 +7,21 @@
 
 
 def test_user_read_permissions(
-    flask_app_client, regular_user, readonly_user, db, test_clone_submission_data
+    flask_app_client, researcher_1, readonly_user, db, test_clone_submission_data
 ):
     # Clone as the regular user and then try to reread as both regular and readonly user,
     # read by regular user should succeed, read by readonly user should be blocked
```

Comment:
Unlikely to confuse, but the comment still refers to regular user instead of researcher 1.